packages/manager/modules/vps/src/dashboard/tile/configuration/translations/Messages_fr_FR.json

```
@@ -15,5 +15,8 @@
   "vps_dashboard_tile_configuration_upgrade_confirmation_title": "Votre modification a bien été prise en compte.",
   "vps_dashboard_tile_configuration_upgrade_confirmation_details": "Vous allez recevoir un e-mail de confirmation dans un délai maximal de deux heures une fois les changements effectifs sur votre VPS. Un redémarrage de votre machine sera nécessaire pour profiter de votre nouvelle configuration.",
   "vps_dashboard_tile_configuration_upgrade_confirmation_track_order": "Suivre ma commande",
-  "vps_dashboard_tile_configuration_upgrade_complete_reboot_required": "L‘ajout de ressources sur votre VPS a bien été effectué. Veuillez redémarrer votre machine afin d‘activer sa nouvelle configuration."
+  "vps_dashboard_tile_configuration_upgrade_complete_reboot_required": "L‘ajout de ressources sur votre VPS a bien été effectué. Veuillez redémarrer votre machine afin d‘activer sa nouvelle configuration.",
+  "vps_dashboard_tile_configuration_upgrade_wrong_nic_cpu": "Pour plus de vCores, merci de contacter le NIC admin : {{ nichandle }}.",
```

May be better if we said this :
```suggestion
  "vps_dashboard_tile_configuration_upgrade_wrong_nic_cpu": "Pour plus de vCores, merci de contacter l'administrateur : {{ nichandle }}.",
```

---

Maybe in second time, the first priority is unblock the tech customer